test/test-helpers.ts

```diff
@@ -201,3 +204,11 @@ export async function signStates(
   const promises = wallets.map(async (w, i) => await sign(w, stateHashes[whoSignedWhat[i]]));
   return Promise.all(promises);
 }
+
+export function transformInputData(object, addresses) {
```

Comment:
Maybe this could be called `convertAddresses` or something?

Reply:
I went for `replaceAddresses` in ec7aaad